modules/nvidia_plugin/src/cuda/node_params.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2020-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cuda_runtime_api.h>
+
+#include <vector>
+
+namespace CUDA {
+
+struct NodeParams {
+    NodeParams(void* kernel, dim3 gridDim, dim3 blockDim) {
+        knp_.func = kernel;
```

Comment:
[C.49](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c49-prefer-initialization-to-assignment-in-constructors): Prefer initialization to assignment in constructors (or in-class initializers)

Reply:
Done